### PR TITLE
Contribution: Education - Translation into german of: "Data Mine Chapter IV & V"

### PR DESCRIPTION
**Pull request by NicN11**

### CODE REVIEW

File: Projects/Foundations/Schemas/Docs-Topics/H/Heikin/Heikin-Data-Mine/heikin-data-mine-003-heikin-ashi-supertrend-heikin-indicator.json [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+                "updated": 1643521546610
+            },
+            {
+                "language": "DE",
+                "text": "Erstellen Sie den Supertrend Indikator, indem Sie die Heikin Ashi Candles als Grundlage für seine Berechnungen verwenden.",
+                "updated": 1644726217034
+            }
+        ]
+    },
+    "paragraphs": [
```
Comment (gitforkgit) [inline Projects/Foundations/Schemas/Docs-Topics/H/Heikin/Heikin-Data-Mine/heikin-data-mine-003-heikin-ashi-supertrend-heikin-indicator.json:21]: Did you make the files that are shown completely green or did you just translate it into german?

**Review by gitforkgit — COMMENTED** [review]
Did you make the files that are shown completely green or did you just translate it into german?

**Comment by NicN11** [comment]
What do you mean with completely green?

**Comment by gitforkgit** [comment]
@NicN11  Completely green means that the file before didn`t exist. Did you write the article and then translate it or did you just translate it?

**Comment by NicN11** [comment]
@gitforkgit translated the docs from english to german.  After the docs.save, the new files were in the new directories.  It was the same with the previous doc.  With the current translation, I'm working ahead, it's the same as always.  Have no idea why that is.

**Review by gitforkgit — APPROVED** [review]
I have merged this pull request now. Please go trough the files that where shown completely green and use the "go to file" function on the repository site to look if the files now exist twice.

**Pull request merged**